app/src/main/res/layout/item_tab.xml

```diff
@@ -67,24 +64,20 @@
             android:background="?selectableItemBackground"
             android:contentDescription="@string/closeContentDescription"
             android:scaleType="center"
-            android:src="@drawable/ic_close_tab"
-            android:tint="?normalTextColor"
+            android:src="@drawable/ic_close_24"
```

Comment:
I think this icon should be `Close-24-small` that need to be imported in the project

Reply:
All good!